vendor-extra/LiberoPageBundle/src/DependencyInjection/LiberoPageConfiguration.php

```diff
@@ -72,14 +75,12 @@ private function getContentPagesDefinition() : ArrayNodeDefinition
         /** @var ArrayNodeDefinition $pagesNode */
         $pagesNode = $builder->root('content');
         $pagesNode
+            ->normalizeKeys(false)
             ->arrayPrototype()
                 ->children()
                     ->scalarNode('path')
                         ->isRequired()
                     ->end()
-                    ->scalarNode('content_service')
-                        ->isRequired()
-                    ->end()
```

Comment:
This change should be extracted. This assumes there's 1-to-1 between a content service and a content page (can't believe you would duplicate them?), and removes the name in favour of the service id. Using that in the route allows us to generate the route name from elsewhere, like https://github.com/libero/browser/blob/137582fa1f021ede7ed22c6075178eab604ede9a/vendor-extra/LiberoPageBundle/src/EventListener/BuildView/ItemTeaserHrefListener.php#L39

Reply:
https://github.com/libero/browser/pull/65